parachain/pallets/ethereum-beacon-client/fuzz/README.md

```diff
@@ -0,0 +1,21 @@
+# Beacon client fuzz tests
+
+This crate contains fuzz tests for the three beacon client extrinsics.
+
+# Installation
+
+```
+cargo install cargo-fuzz
```

Comment:
Would be better also add to init script https://github.com/Snowfork/snowbridge/blob/b2b2e9c9e131b9643268c512c8972f597a29559f/scripts/init.sh#L15-L16